packages/@ember/object/compat.ts

```diff
@@ -2,15 +2,21 @@ import { Meta } from '@ember/-internals/meta';
 import {
   Decorator,
   DecoratorPropertyDescriptor,
+  descriptorForProperty,
   isElementDescriptor,
   setClassicDecorator,
 } from '@ember/-internals/metal';
 import { assert } from '@ember/debug';
 import { consumeTag, tagFor, track, UpdatableTag, updateTag } from '@glimmer/validator';
 
-let wrapGetterSetter = function(_target: object, key: string, desc: PropertyDescriptor) {
+let wrapGetterSetter = function(target: object, key: string, desc: PropertyDescriptor) {
   let { get: originalGet } = desc;
 
+  assert(
+    'You attempted to use @computed on a property that already has been decorated with either @computed or @tracked. @dependentKeyCompat is only necessary for native getters that are not decorated with @computed.',
```

Comment:
@pzuraq do you mean "You attempted to use @dependentKeyCompat..." here?

Reply:
Good catch, fixed!